content/wiki/canonical_lints.md

```diff
@@ -75,14 +75,16 @@ clippy.wildcard_dependencies = "warn"
 ## `lib.rs`
 
 ```rust
-// LINEBENDER LINT SET - lib.rs - v2
+// LINEBENDER LINT SET - lib.rs - v3
 // See https://linebender.org/wiki/canonical-lints/
-// These lints aren't included in Cargo.toml because they
-// shouldn't apply to examples and tests
-#![warn(unused_crate_dependencies)]
-#![warn(clippy::print_stdout, clippy::print_stderr)]
+// These lints aren't included in Cargo.toml because they shouldn't apply to examples and tests.
+#![cfg_attr(not(test), warn(unused_crate_dependencies))]
+#![cfg_attr(not(test), warn(clippy::print_stderr, clippy::print_stdout))]
```

Comment:
Fwiw I also slightly prefer the stdout/stderr ordering; when describing them as a pair, I'd always expect them to be in that order. That one is not a strongly held position

Reply:
I was merely following the directions on this very same page:
> If you think a new lint should be added to Linebender projects, add it to this file in alphabetical order

However I didn't write that rule and don't feel strongly about enforcing it. I'll let someon else do that if they care enough. Reverted the change.